src/components/ProductTours/AdminOnboardingTours/tests/AdminOnboardingTours.test.jsx

```diff
@@ -26,29 +49,52 @@ const mockStore = configureStore([]);
 describe('AdminOnboardingTours', () => {
   let store;
 
+  const mockedInsights = {
+    learner_progress: {
+      enterprise_customer_uuid: 'aac56d39-f38d-4510-8ef9-085cab048ea9',
+    },
+    learner_engagement: {
+      enterprise_customer_uuid: 'aac56d39-f38d-4510-8ef9-085cab048ea9',
+    },
+  };
+
+  const slug = 'Stark Industries';
+  const enterpriseAdminUuid = 'test-uuid';
+
   beforeEach(() => {
     store = mockStore({
       portalConfiguration: {
         enableLearnerPortal: true,
-        enterpriseSlug: 'test-enterprise',
+        enterpriseSlug: slug,
         enterpriseFeatures: {
           enterpriseAdminOnboardingEnabled: true,
         },
       },
+      dashboardInsights: {
+        loading: false,
+        insights: mockedInsights,
+      },
+      enterpriseCustomerAdmin: {
+        uuid: enterpriseAdminUuid,
+      },
     });
   });
 
   const defaultProps = {
     isOpen: true,
     onClose: jest.fn(),
-    enterpriseSlug: 'test-enterprise',
+    setTarget: jest.fn(),
     targetSelector: 'learner-progress-sidebar',
   };
 
   const renderComponent = (props = {}) => {
     const finalProps = { ...defaultProps, ...props };
     return render(
       <Provider store={store}>
+        <p id={targets[0]}>Step 1</p>
```

Comment:
Omit the '#' when setting the test element ids.
```suggestion
        <p id={targets[0].slice(1)}>Step 1</p>
```